.vib/gitea/goss/goss.yaml

```diff
@@ -0,0 +1,24 @@
+command:
+  check-user-info:
+    exec: id
+    exit-status: 0
+    stdout:
+      - uid={{ .Vars.containerSecurityContext.runAsUser }}
+      - /groups=.*{{ .Vars.podSecurityContext.fsGroup }}/
+file:
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.automountServiceAccountToken }}
+    filetype: directory
+    mode: "3777"
```

Comment:
It seems the SA token is mounted given if the custom serviceAccount is not being created because it uses the default one.

We have been adding this test only when a SA is created by the chart, but we may have to change this approach and evaluate when, if necessary, we should include it. WDYT?

Reply:
The chart should create the SA, fixed it in the values.yaml file